clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/session/DistributableSession.java

```diff
@@ -258,4 +258,9 @@ private void validate(HttpServerExchange exchange, Session<LocalSessionContext>
             throw UndertowClusteringLogger.ROOT_LOGGER.sessionIsInvalid(session.getId());
         }
     }
+
+    private BatchContext resumeBatch() {
+        Batch batch = (this.batch.getState() != Batch.State.CLOSED) ? this.batch : null;
+        return this.manager.getSessionManager().getBatcher().resumeBatch(batch);
```

Comment:
@pferraro I presume that a null batch is not a problem?

Reply:
Correct.
https://github.com/wildfly/wildfly/blob/master/clustering/ee/infinispan/src/main/java/org/wildfly/clustering/ee/infinispan/InfinispanBatcher.java#L140